src/tests/services/moviedbService.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import {render} from '@testing-library/react'
+import {act} from 'react-dom/test-utils'
+
+import moviedbService from '../../services/moviedbService'
+
+
+describe('moviedbService', () => {
+
+  const mockSuccessResponse = {results: [{title: 'Godzilla vs. Kong', overview: 'Em uma época em que os monstros andam na Terra, a luta da humanidade por seu futuro coloca Godzilla e Kong em rota de colisão que verá as duas forças mais poderosas da natureza no planeta se confrontarem em uma batalha espetacular para as idades. Enquanto Monarch embarca em uma missão perigosa em terreno desconhecido e descobre pistas sobre as origens dos Titãs, uma conspiração humana ameaça tirar as criaturas, boas e más, da face da terra para sempre.', vote_average: 8.3, poster_path: '/klAIFewuqcyEmH1SMtR2XbMyqzM.jpg', release_date: '2021-03-24'}]};
+
+  global.fetch = jest.fn(() => 
+    Promise.resolve({
+      json: () => Promise.resolve(mockSuccessResponse),
+    })
+  );
+  
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it('Calls correct URI fetch', async () => {
```

Comment:
nit (muito nit): eu prefiro começar a descrição do teste com letra minúscula, assumindo que é o meio da frase. Do jeito que tá parece que a frase é "it Calls correct URI" ao invés de "it calls correct URI".

Reply:
butei tudo minúsculo ✌️